docs/cave/d2d-perspective.md

```diff
@@ -57,17 +59,37 @@ From the above menu select **Show Map Legends** and watch the Resource Stack sho
 
 ![image](../images/maplayers.png)
 
+### Sample Loaded Resources
+
+Most data types have a right-click menu option for reading out the pixel value, displayed as multi-line text for multiple resources.  This can be toggled on and off by selecting the **Sample** option in the Display Menu.
+
+![image](../images/ui4fNI3X0C.gif)
+
 ### Toggle 2 or 4-Panel Layout
 
-Right-click hold in the view and select **Two Panel Layout** or **Four Panel Layout** to create duplicates of the current view (note that any data loaded to the *view* will be loaded to *both displays within the view*).
+Right-click hold in the view and select **Two Panel Layout** or **Four Panel Layout** to create duplicates of the current view.
+
+!!! note "Notice the readout is at the same position in both panels.  Any mouse movement made on one panel will be made on the other."
+
+![2-panel](../images/2panelReadout.png)
+
+By default, loading any data will load that data onto **both** panels.  However, there is the option to specify which panel you would like to load data into, which can be useful if you want to have different data in each of the panels.  To access this option, simple hold-right click to pull up the Display menu and choose **Load to This Panel** as shown below:
+
+![select panel load](../images/loadToThisPanel.png)
+
+Now, a yellow **L** will appear in the lower left hand corner of the panel you selected to load data to.  When data is loaded from the menus it will only load to the display desginated with the L. 
+Switch back to loading in both panels, by using the **Load to All Panels** option in the Display Menu.
+
+![single panel load](../images/singlePanelLoad.png)
 
 From this multi-pane display, hold-right-click again and you will see the **Single Panel Layout** option to switch back to a standard view (defaulting to the left of two, and top-left of four).
 
-### Sample Loaded Resources
+### Unload Data
 
-Most data types have a right-click menu option for reading out the pixel value, displayed as multi-line text for multiple resources.  This can be toggled on and off by selecting the **Sample** option in the Display Menu.
+Select **Unload All Products** to remove all loaded data products from the display and start fresh.
+Select **Unload Graphics** to remove all but the numerical products.
```

Comment:
Hmmm, the label "Unload Graphics" is sort of misleading (not your fault). I referenced the actual user manual to see how they explained it (https://www.unidata.ucar.edu/software/awips2/doc/CAVE_D2D_Manual.pdf - search for "Unload Graphics")
We could say "to remove all but the loaded images". 
Because for example you can specify to load a gridded/satellite product as an image or a contour.

Reply:
Okay cool, I pushed another change so if you check out the "files changed" it should be slightly different now.